packages/web-components/fast-foundation/src/number-field/README.md

```diff
@@ -3,6 +3,7 @@ id: number-field
 title: fast-number-field
 sidebar_label: number-field
 custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-components/fast-foundation/src/number-field/README.md
+description: fast-number-field is a web component implementation of a text field.
```

Comment:
I think this should say "number input" or something like that at the end.

Reply:
Thank you @EisenbergEffect. 

Description updated to:

"fast-number-field is a web component implementation of a text field number input."